tests/feed/test_feed.py

```diff
@@ -77,3 +77,29 @@ def test_posts_order_in_dict_by_creation_date(self, num_of_posts, persist_user,
 
         posts_for_user = list(PostManager.get_posts_for_user(persist_user).keys())
         assert all(earlier.date <= later.date for earlier, later in zip(posts_for_user, posts_for_user[1:]))
+
+
+@pytest.mark.django_db
+class TestPostsListView:
+    def test_posts_list_view_loaded_authorized(self, authorized_client):
+        post_list_response = authorized_client.get('/feed/list/')
+        assert post_list_response.status_code == 200
```

Comment:
I'd like to see a test that check for the existence of an expected post.

Reply:
Isnt it the last test here? I am creating a post and check that it exist

Reply:
The last test seems to check that the Post object was created, I want you to check a post created actually appears in the posts view.

Reply:
done, is that what you meant?

Reply:
removed duplicate test